content/metadata.yaml

```diff
@@ -500,7 +500,7 @@ authors:
     contributions:
       - Writing - Original Draft
     code of conduct:
-      confirmed: No
+      confirmed: yes
     affiliations:
       - Institute of Translational Medicine and Therapeutics, Perelman School of Medicine, University of Pennsylvania, Philadelphia, Pennsylvania, United States of America
     coi:
```

Comment:
coi
lastapproved: !!str 2020-11-9

Reply:
So I think you need to go in and edit the `metadata.yml` file in the same way you updated the `confirmed: yes` text and commit and push those changes to your `patch-8` branch. Does that make sense?